src/store/conversations.js

```diff
@@ -21,6 +21,15 @@
  */
 import Vue from 'vue'
 
+const getDefaultState = () => {
+	return {
+		conversations: {
+		},
+		conversationsNames: {
```

Comment:
What would be the data inside this?
Using token as indexes?
What is the purpose of this conversationsNames?

Reply:
Ehrm, currenly nothing